src/localizations/no_nb.json

```diff
@@ -41,8 +41,8 @@
     "api_received_overview_text": "Her finner du oversikt over de API-tilganger din virksomhet har blitt tildelt av andre virksomheter.",
     "cancel": "Avbryt",
     "give_access_to_new_api": "Gi tilgang til nytt API",
-    "new_org_accordion_content_text": "Velg hvilke brukere som skal få tilgang ved å klikke pluss-tegnet. Du kan også legge til ny virksomhet ved å benytte søkefeltet.",
-    "businesses_previously_delegated_to": "Virksomheter det tidligere er delegert til:",
+    "new_org_content_text": "Velg hvilke brukere som skal få tilgang ved å klikke pluss-tegnet. Du kan også legge til ny virksomhet ved å benytte søkefeltet.",
+    "businesses_previously_delegated_to": "Tidligere delegerte virksomheter:",
```

Comment:
It will fix a css related problem where the long wording makes the accordions seem unaligned. Also I don't think anyone will see a different in the wording when they don't have anything to compare with like you have here.